src/host/services/IMUService.cpp

```diff
@@ -26,38 +26,199 @@
 #include "common/signalk/SKUpdateStatic.h"
 #include "common/signalk/SKUnits.h"
 #include "IMUService.h"
+#include "host/util/PersistentStorage.h"
 
 void IMUService::setup() {
+  _timeSinceLastSetOffset = 0;
+  _offsetRoll  = 0.0;
+  _offsetPitch = 0.0;
+  // Default mounting position: VerticalStbHull
+  uint8_t axisConfig = 0b00001001;
+  uint8_t signConfig = 0b00000000;
+
+  if (_config.mounting == VerticalTopToBow) {
```

Comment:
same here we should use an enum to make sure we cover all the cases.

Reply:
now switch statement and yet non configured positions commented out in enum